7.0.0/postgresql/shared_db_cleanup.sh

```
@@ -0,0 +1,132 @@
+    fi
+
+    echo "Completed deletion for TENANT_ID=$TENANT_ID."
+  else
+    echo "Skipping invalid or empty line: TENANT_ID=$TENANT_ID, ORG_UUID=$ORG_UUID"
+  fi
+done < "$EXPORT_FILE"
+
+echo "Data deletion process completed. Summary available at $SUMMARY_FILE."
+
+# Unset PGPASSWORD to avoid leaving it in the environment
+unset PGPASSWORD
```

Shellcheck (0.11.0)

[error] Literal carriage return. Run script through tr -d '\r' .

(SC1017)